ngraph/core/src/op/negative.cpp

```diff
@@ -10,14 +10,22 @@
 using namespace std;
 using namespace ngraph;
 
-constexpr NodeTypeInfo op::Negative::type_info;
+NGRAPH_RTTI_DEFINITION(op::v0::Negative, "Negative", 0, util::UnaryElementwiseArithmetic);
 
 op::Negative::Negative(const Output<Node>& arg)
     : UnaryElementwiseArithmetic(arg)
 {
     constructor_validate_and_infer_types();
 }
 
+void op::Negative::validate_and_infer_types()
+{
+    NGRAPH_OP_SCOPE(v0_Negative_validate_and_infer_types);
+    auto input_et = get_input_element_type(0);
+    NODE_VALIDATION_CHECK(this, input_et.is_signed(), "Input type has to be signed");
```

Comment:
As I can see this check breaks few tests resulting into backward incompatible changes: https://github.com/openvinotoolkit/openvino/pull/5955/checks?check_run_id=2791739944

I agree this is a bug and should be addressed, but we can do it in a separate PR as it involves changes into OMZ onnx models. Could you please comment on this? @lazarevevgeny

Reply:
Removed assert as it introduces backward incompatibility